src/targets/gpu/kernels/include/migraphx/kernels/integral_constant.hpp

```diff
@@ -73,10 +73,10 @@ MIGRAPHX_INTEGRAL_CONSTANT_BINARY_OP(!=)
 MIGRAPHX_INTEGRAL_CONSTANT_BINARY_OP(&)
 MIGRAPHX_INTEGRAL_CONSTANT_BINARY_OP(^)
 MIGRAPHX_INTEGRAL_CONSTANT_BINARY_OP(|)
-MIGRAPHX_INTEGRAL_CONSTANT_BINARY_OP(&&)
-MIGRAPHX_INTEGRAL_CONSTANT_BINARY_OP(||)
+MIGRAPHX_INTEGRAL_CONSTANT_BINARY_OP(and)
```

Comment:
Are these supposed to be changed?

Reply:
Yes, although the cppcheck rule will not catch this right now since it uses the preprocessed tokens. When I do this with the addon this will need to be changed since the macro will expand to using those operators. I dont see an issue with changing this now.